### PR TITLE
Fix dates for the NumPy Newcomers' meeting on the calendar

### PR DESCRIPTION
**Pull request by melissawm**
Meetings were scheduled with the wrong date.

### CODE REVIEW

File: calendars/numpy.yaml [hunk 1]
```diff
@@ -46,8 +46,8 @@ events:
     description: |
       The fortnightly NumPy call for newcomers.
 
-    begin: 2022-01-17 16:00:00 +00:00
-    end: 2022-01-17 17:00:00 +00:00
+    begin: 2022-01-10 16:00:00 +00:00
+    end: 2022-01-10 17:00:00 +00:00
```
Comment (rossbar) [inline calendars/numpy.yaml:50]: 01-10-22 is still a Monday, should this be 01-13-22 (i.e. a Thursday) instead?

Comment (rossbar) [reply calendars/numpy.yaml:50]: Or maybe it was supposed to be 2022-02-10, which was a Thursday :joy:

Comment (melissawm) [reply calendars/numpy.yaml]: I obviously messed up the dates - sorry 😅

**Review by jarrodmillman — APPROVED** [review]

**Review by rossbar — APPROVED** [review]
Works for me now in the preview - thanks @melissawm !

**Pull request merged**